src/components/PostCard.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import clsx from 'clsx';
+import Card from '@material-ui/core/Card';
+import CardHeader from '@material-ui/core/CardHeader';
+import CardContent from '@material-ui/core/CardContent';
+import CardActions from '@material-ui/core/CardActions';
+import Collapse from '@material-ui/core/Collapse';
+import Avatar from '@material-ui/core/Avatar';
+import IconButton from '@material-ui/core/IconButton';
+import Typography from '@material-ui/core/Typography';
+import { red } from '@material-ui/core/colors';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import MoreVertIcon from '@material-ui/icons/MoreVert';
+import ThumbUpAltIcon from '@material-ui/icons/ThumbUpAlt';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: '100%'
```

Comment:
```suggestion
    width: '90%',
    marginBottom: 20
```

Reply:
Better off changing the size of the grid that envelops them instead since it'll be more responsive